### PR TITLE
Create link_multistage_frame_io.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add new coverage for abused frame.io as landing pages

# Associated samples

- [Sample 1](https://platform.sublime.security/messages/2db9d38cc00a5b52e25db4a62691509a88c9edbfd0ed8ffa78c8c847f916fa93)
- [Sample 2](https://platform.sublime.security/messages/a3c20c70fe78c83f1addaf8cbc6b04dd6aa3b3b1ede874aa2d92a1486b6d2650)

## Associated hunts

- [Hunt 1](https://platform.sublime.security/hunts/15b5d514-7644-42ec-be26-7883ce461d5f)

### CODE REVIEW

**Comment by zoomequipd** [comment]
No hits in shared EML.  Will continue to monitor for another week.

**Comment by zoomequipd** [comment]
still no hits in shared EMLs, ready for review

File: detection-rules/link_multistage_frame_io.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  and sender.email.domain.domain == "frame.io"
+  // these messages contain no email address of the actual sender
+  // so sender profile won't be interesting
+  // however the subject and sender display names do contain the name of the frame.io account
+  // which sent the share
+  
+  // negate where internal employees might have sent the message
+  and not any($org_display_names, strings.istarts_with(subject.subject, .))
+  
+  // the subject conatins the name of the file that has been shared
```
Comment (aidenmitchell) [inline detection-rules/link_multistage_frame_io.yml:16]:
```suggestion
  // the subject contains the name of the file that has been shared
```

File: detection-rules/link_multistage_frame_io.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
+name: "Link: Multistage Landing - Abused Adobe frame.io"
+description: "The detection rule matches on message groups which make use of Adobe's frame.io as a landing page. The landing page contains links which are newly registered, use free file or subdomain hosts, url shortners or when visited are phishing pages, lead to a captcha or redirect to a top website"
```
Comment (aidenmitchell) [inline detection-rules/link_multistage_frame_io.yml:2]:
```suggestion
description: "The detection rule matches on message groups which make use of Adobe's frame.io as a landing page. The landing page contains links which are newly registered, use free file or subdomain hosts, url shortners or when visited are phishing pages, lead to a captcha or redirect to a well-known domain, seen in evasion tactics."
```

File: detection-rules/link_multistage_frame_io.yml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                      and .scan.qr.url.domain.root_domain not in (
+                        'frame.io',
+                        'f.io',
+                        'onetrust.com'
+                      )
+                    )
+                    // some samples have a pdf uploaded that contains a Sharepoint File Share lure.
+                    // we can use ocr to detect this
+                    or strings.icontains(.scan.ocr.raw,
+                                         'This email contains a secure lik to sharepoint'
```
Comment (aidenmitchell) [inline detection-rules/link_multistage_frame_io.yml:155]: Is this a typo or an artifact of OCR?

**Pull request merged**